threedi_plugin.py

```diff
@@ -78,10 +78,9 @@ def __init__(self, iface):
         self.toolbar_animation = self.iface.addToolBar("ThreeDiAnimation")
         self.toolbar_animation.setObjectName("ThreeDiAnimation")
 
-        self.timeslider_widget = TimesliderWidget(
-            self.toolbar_animation, self.iface, self.ts_datasources
-        )
+        self.timeslider_widget = TimesliderWidget(self.iface, self.ts_datasources)
         self.lcd = QLCDNumber()
+        self.lcd.setToolTip("Time (days hours:minutes)")
```

Comment:
Ik zou er "format" bij zetten. Zoiets?
```suggestion
        self.lcd.setToolTip("Time format: \"days hours:minutes\"")
```